config/environment.rb

```diff
@@ -1,6 +1,7 @@
 # Load the Rails application.
 require_relative 'application'
-require Rails.root.join(Karafka.boot_file)
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+require Rails.root.join(Karafka.boot_file)
```

Comment:
this needs to be added AFTER the rails framework is loaded. That's why Karafka did not see the consumers that are part of Rails boot loading